### ISSUE TITLE
Reverse naming for _edited to _original

### ISSUE THREAD

**Issue by wormiedk** [RhetTbull/osxphotos#263]
I would like to move all my livephotos and originals to subfolders after exporting in order to make my main folder more "clean". In order to do that it would be great if _edited photos were just called by their name and that the original photo instead had an _original suffix.

**Comment by RhetTbull** [comment]
I think I can implement this fairly easily with the new boolean types I recently added to the template system.  For example, something like this:

`osxphotos export /path/to/export --filename "{original_filename}{edited?,_original}" --edited-suffix ""`

the `{edited?,_original}` is read as "if photo is edited, add no suffix and if not, add _original suffix".  Then the `--edited-suffix ""` means don't add any suffix for edited photos.  

I can probably get this added in the next couple of weeks.

**Comment by RhetTbull** [comment]
> I would like to move all my livephotos and originals to subfolders after exporting 

Also be advised that this will cause `--update` to re-export all those photos if you run osxphotos again.

**Comment by RhetTbull** [comment]
I added `--original-suffix` to release [v0.36.25](https://github.com/RhetTbull/osxphotos/releases/tag/v0.36.25).  You can achieve what you specified using `--original-suffix "_original" --edited-suffix ""`

**Comment by wormiedk** [comment]
This works however what I would really like is for the photos only to be named _original if there is an edited version available. Otherwise suffix should not be added... In that way I can ask plex to ignore files with _original when viewing photos and still se all photos

**Comment by RhetTbull** [comment]
Implementing the `{edited?}` template I originally proposed above will not be simple because it would move evaluation of the template code to another place in the export process.  However I could easily add a template that is True if photo has edits (regardless of whether evaluating the original or edited version) and False otherwise.

Thus if I added a new `{hasedits?}` template that is True if photo has edits and False otherwise and then also make `--original-suffix` and `--edited-suffix` templates instead of simple strings, you could then do this:

`osxphotos export /path/to/export --original-suffix "{hasedits?_original,}" --edited-suffix ""`

**Comment by wormiedk** [comment]
That would be great! Also for live photos 👍

**Comment by RhetTbull** [comment]
I have this working, need to add tests before pushing a release.  I decided to call the new template `{edited?}` as that matches the existing query option `--edited`.  If the photo has been edited, this resolves to True, otherwise False.  So I think this will now do what you want:

`osxphotos export ~/Desktop/export -V --original-suffix "{edited?_original,}" --edited-suffix ""`

```
...
Exporting wedding.jpg (E9BC5C36-7CD1-40A1-A72B-8B8FAC227D51.jpeg) as wedding_original.jpg
Exporting edited version of wedding.jpg (E9BC5C36-7CD1-40A1-A72B-8B8FAC227D51.jpeg) as wedding.jpeg
Exported /Users/rhet/Desktop/export/wedding_original.jpg
Exported /Users/rhet/Desktop/export/wedding.jpeg
Exporting IMG_1994.JPG (A92D9C26-3A50-4197-9388-CB5F7DB9FA91.jpeg) as IMG_1994.JPG
Exported /Users/rhet/Desktop/export/IMG_1994.JPG
Exported /Users/rhet/Desktop/export/IMG_1994.cr2
...
```
I'll work on the tests and other boolean template values later today.

**Comment by RhetTbull** [comment]
This is fixed in [v0.38.4](https://github.com/RhetTbull/osxphotos/releases/tag/v0.38.4).  I've not added all the boolean template types yet but did add {edited} so you can do:

`osxphotos export ~/Desktop/export -V --original-suffix "{edited?_original,}" --edited-suffix ""`

**Comment by wormiedk** [comment]
Works great in conjuction with a .plexignore file containing *_original* now my whole family can access our photo library! Thanks a lot for your great software. How do I buy you a pint? 👍
